react/FilterNavigator.js

```diff
@@ -137,7 +137,7 @@ const FilterNavigator = ({
         </div>
       ) : (
         <Fragment>
-          <div className={filterClasses}>
+          <div className={handles.filtersWrapper}>
```

Comment:
The previous classes should be kept in place:

```suggestion
          <div className={`${filterClasses} ${handles.filtersWrapper}`}>
```